.pathplanner/settings.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Is this file autogenerated?

Reply:
This file seems like it might be important in order to correctly visualize paths. It is autogenerated after we enter these values into the pathplanner application

Reply:
If we keep this file, do we not have to re-enter the values into PathPlanner? We might want to keep it then. You could an add exception to .gitignore like this:
```
.pathplanner
!.pathplanner/settings.json
```